docs/learn-faq.md

```diff
@@ -8,15 +8,15 @@ _This FAQ focuses on technical questions for users interested in developing appl
 
 ### How many validators will be online at Mainnet launch?
 
-Somewhere between 50 - 100, increasing to 1000 or so later on. [Source](https://youtu.be/IRc5Jma_eH8?t=1630)
+Somewhere between 50 - 100, increasing to 1000 or so later on. [Source](https://youtu.be/IRc5Jma_eH8?t=1630)  There are currently 160 validators in the active validator set on the Kusama network.
```

Comment:
I would shy away from using fixed numbers here. That's what #177 should help with.

Reply:
Fixed